src/utils/components/DiskModal/components/StorageProfileSettings/VolumeMode.tsx

```diff
@@ -1,58 +1,96 @@
-import React, { FC, useEffect, useMemo } from 'react';
-import { Controller, useFormContext } from 'react-hook-form';
+import React, { FC, useCallback, useEffect } from 'react';
+import { useFormContext } from 'react-hook-form';
+import { Trans } from 'react-i18next';
+import { Link } from 'react-router-dom-v5-compat';
+import { uniq } from 'lodash';
```

Comment:
```suggestion
import uniq from 'lodash/uniq';
```

Reply:
i would fix that in another pr

Reply:
@upalatucci 
We could. However note that lodash as whole is a dependency of our core libs including:
1. `@openshift-console/dynamic-plugin-sdk`
2. `@patternfly` libs (charts,table)